### ISSUE TITLE
The list is not accessible for a11y users

### ISSUE THREAD

**Issue by spanjwani2106** [zurb/tribute#572]
As it is an accessibility(a11y) issue, you need to use NVDA Speech Viewer. 
Steps:
1. Type @ on the editor, move through the list using down arrow keys.
2. You will notice that the speech viewer doesn't read the user name it reads @ on each press of down key.

Expected - 
It should read the name of the user on which the user moved to using arrow key.

![image](https://user-images.githubusercontent.com/73885473/97979481-3f3baa80-1df5-11eb-8c53-ddbfe44d3969.png)
